src/main/java/graphql/schema/PropertyDataFetcher.java

```diff
@@ -46,7 +47,7 @@
  * @see graphql.schema.DataFetcher
  */
 @PublicApi
-public class PropertyDataFetcher<T> implements DataFetcher<T> {
+public class PropertyDataFetcher<T> implements DataFetcher<T>, TrivialDataFetcher<T> {
```

Comment:
This can be simply be

     public class PropertyDataFetcher<T> implements TrivialDataFetcher<T>